apitest/src/test/java/bisq/apitest/method/wallet/BtcWalletTest.java

```diff
@@ -92,6 +93,38 @@ public void testFundAlicesBtcWallet(final TestInfo testInfo) {
                 formatBtcBalanceInfoTbl(btcBalanceInfo));
     }
 
+    @Test
+    @Order(3)
+    public void testAliceSendBTCToBob(TestInfo testInfo) {
+        String bobsBtcAddress = getUnusedBtcAddress(bobdaemon);
+        log.debug("Sending 5.5 BTC From Alice to Bob @ {}", bobsBtcAddress);
+
+        sendBtc(alicedaemon, bobsBtcAddress, "5.50", "100");
+        genBtcBlocksThenWait(1, 3000);
+
+        BtcBalanceInfo alicesBalances = getBtcBalances(alicedaemon);
+
+        log.debug("{} Alice's BTC Balances:\n{}",
+                testName(testInfo),
+                formatBtcBalanceInfoTbl(alicesBalances));
+        bisq.core.api.model.BtcBalanceInfo alicesExpectedBalances =
+                bisq.core.api.model.BtcBalanceInfo.valueOf(700000000,
+                        0,
+                        700000000,
+                        0);
+        verifyBtcBalances(alicesExpectedBalances, alicesBalances);
+
+        BtcBalanceInfo bobsBalances = getBtcBalances(bobdaemon);
+        log.debug("{} Bob's BTC Balances:\n{}",
+                testName(testInfo),
+                formatBtcBalanceInfoTbl(bobsBalances));
+        // We cannot (?) predict the exact tx size and calculate how much in tx fees were
```

Comment:
I think so too, but the `sendbtc` tx size randomly varies between two distinct values (219 bytes or 220 bytes) from test run to test run, hence the available balance range [1549978000, 1549978100].
Is that a bug?  
Should I be more specific about this in that test case's comment?

Reply:
If the same address is used every time it should be deterministic, but if it's sending to different addresses due to some non deterministic events within the test it could result in different size txs.

Reply:
Commit 9250845a66bc7cda2c369806bf0949537bb5ac18 changes the comment:
```
// The sendbtc tx weight and size randomly varies between two distinct values
// (876 wu, 219 bytes, OR 880 wu, 220 bytes) from test run to test run, hence
// the assertion of an available balance range [1549978000, 1549978100].
```